File: snippets/typescript/recommend_point_groups.ts
import { QdrantClient } from "@qdrant/js-client-rest";

const client = new QdrantClient({ host: "localhost", port: 6333 });

client.recommendPointGroups("{collection_name}", {
    positive: [100, 231],
    negative: [718],
    group_by: "document_id",
    limit: 3,
    group_size: 2,
});

File: snippets/python/recommend_point_groups.py
from qdrant_client import QdrantClient, models

client = QdrantClient(url="http://localhost:6333")

client.recommend_groups(
    collection_name="{collection_name}",
    positive=[100, 231],
    negative=[718],
    group_by="document_id",
    limit=3,
    group_size=2,
)
